.ci/bump-cs-version-in-esclipsecs-plugin.sh

```diff
@@ -10,6 +10,6 @@ fi
 VERSION=$1
 cd eclipsecs-sevntu-plugin
 
-mvn -e --no-transfer-progress versions:set-property -DgenerateBackupPoms=false -Dproperty=checkstyle.version -DnewVersion="$VERSION"
+sed -i -e "/<properties>/,/<\/properties>/ s|<checkstyle.version>.*</checkstyle.version>|<checkstyle.version>$VERSION</checkstyle.version>|g" pom.xml
```

Comment:
What for this block `<properties>/,/<\/properties>` ?

Reply:
> Is it a fancy syntax to explain sed to do change only between such tags ?

Correct. 

 - [sed documentation on range addresses](https://www.gnu.org/software/sed/manual/html_node/Range-Addresses.html)
 - https://stackoverflow.com/a/22960617/9553927